eslint.config.mjs

```diff
@@ -0,0 +1,102 @@
+import eslintConfig from '@yarnpkg/eslint-config';
+import reactEslintConfig from '@yarnpkg/eslint-config/react';
+
+export default [
+  ...eslintConfig,
+  ...reactEslintConfig,
+
+  {
+    ignores: [
+      `.yarn`,
+
+      `packages/docusaurus/.docusaurus`,
+
+      `packages/yarnpkg-pnp/sources/hook.js`,
+      `packages/yarnpkg-pnp/sources/esm-loader/built-loader.js`,
+
+      `packages/yarnpkg-libzip/sources/libzipAsync.js`,
+      `packages/yarnpkg-libzip/sources/libzipSync.js`,
+
+      // The parsers are auto-generated
+      `packages/yarnpkg-parsers/sources/grammars/*.js`,
+
+      `packages/yarnpkg-core/sources/worker-zip/index.js`,
+      `packages/yarnpkg-cli/bundles/yarn.js`,
+
+      `packages/*/lib`,
+      `packages/*/bin`,
+      `packages/*/build`,
+      `packages/**/*fixtures*`,
+      `packages/yarnpkg-libzip/artifacts`,
+      `packages/plugin-compat/extra/fsevents/fsevents-*.js`,
+
+      // Files generated by Gatsby
+      `packages/gatsby/public`,
+      // Files generated by TypeDoc`,
+      `packages/gatsby/static/api`,
+
+      // Minimize the diff with upstream`,
+      `packages/yarnpkg-pnp/sources/node`,
+      `packages/yarnpkg-pnp/sources/loader/node-options*`
+    ],
```

Comment:
1. The gatsby rules should be safe to remove
2. Is `yarnpkg-cli` the only workspace to generate a `bundle` directory? Because other places just ignores `packages/*/bundle`
3. Should the `yarnpkg-libui` artifacts also be here?
4. Should the these be grouped by package?

Reply:
I removed Gatsby, but other changes will be for another PR (I'd like to keep the changes here as minimal as possible).